src/main/java/com/vandendaelen/handles/functions/handles/GetCurrentDimension.java

```diff
@@ -20,6 +22,7 @@ public boolean impactMoodAndLoyalty() {
 
     @Override
     public MethodResult run(ConsoleTile tardis, IArguments args) {
-        return MethodResult.of(DimensionHelper.getDimensionId(tardis.getCurrentDimension()));
+        final RegistryKey<World> currentDimension = tardis.getCurrentDimension();
+        return MethodResult.of(DimensionHelper.getDimensionId(tardis.getCurrentDimension()), currentDimension.location().getPath());
```

Comment:
What's path ?

Reply:
Path would be "the_nether" or "overworld", so that it doesn't show "minecraft:" in front of it. Though I'm now thinking that excluding the modid may not be the correct idea

Reply:
Oh ok ! Thanks !